packages/IMAPClient-UI.package/ICLoginDialog.class/instance/buildWith..st

```diff
@@ -85,10 +85,19 @@ buildWith: builder
 
 	window := builder build: windowSpec.
 	
+	window on: #keyStroke send: #keyEvent: to: self.
+	self loginMorph: window.
+	
+	self inputCollection: OrderedCollection new.
+	
+	
+	window submorphsDo: [ :m |
+	"add listener to text fields"	
+		(m class == PluggableTextMorphPlus)
+			ifTrue: [self inputCollection add: m.
```

Comment:
I feel like there are a lot of inconsistency regarding leaving spaces in blocks. Sometimes you leave space at the front but not at the end, sometimes you do not leave any space at all.

Reply:
http://live.exept.de/doc/online/english/programming/codingStyle.html

They are always using `do: [:each | each stuff]`. Will stick to that from now on. :)